### PR TITLE
Feature/faq page master rebase

### PR DESCRIPTION
**Pull request by andresgnlez**

### CODE REVIEW

File: app/src/components/Shared/LinkTo.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
```
Comment (tiagojsag) [inline app/src/components/Shared/LinkTo.jsx]: this whole class seems to be dead code

File: app/src/components/FAQ/Accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import LinkTo from './../Shared/LinkTo';
+import { Accordion, AccordionItem } from 'react-sanfona';
+
+import AccordionStyles from '../../../styles/components/FAQ/c-accordion.scss';
+
+class FAQAccordion extends Component {
```
Comment (tiagojsag) [inline app/src/components/FAQ/Accordion.jsx]: Rename ContentAccordion (also rename scss file) and move to shared

File: app/src/components/FAQ/Accordion.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false
```
Comment (tiagojsag) [inline app/src/components/FAQ/Accordion.jsx]: is this used at all?

File: app/src/components/FAQ/Accordion.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  render() {
+    return (
+      <Accordion
+        allowMultiple={false}
+        activeItems={[-1]}
+        className={AccordionStyles['c-accordion']}
+        onChange={this.toggleItemBinded}
```
Comment (tiagojsag) [inline app/src/components/FAQ/Accordion.jsx]: i think this is also dead

File: app/src/components/FAQ.jsx [hunk 5]
```diff
@@ -10,43 +12,17 @@ class FAQ extends Component {
...
-
     return (<div>
       <CoverPrimary
         title="Frequently Asked Questions"
         subtitle="Get answers to commonly asked questions about Global Fishing Watch and commercial fishing."
       />
-      <section>
-        {faqPageContent}
-      </section>
+      {this.props.faqEntries &&
```
Comment (tiagojsag) [inline app/src/components/FAQ.jsx]: check the Blog page and use the same "loading" (yet unstyled) if props.faqEntries is empty

File: app/styles/index.scss [hunk 6]
```diff
@@ -1,16 +1,18 @@
 @charset "utf-8";
+
 /*Base*/
 @import 'base';
 @import 'base/normalize';
 @import 'base/slick';
-@import 'base/slick';
```
Comment (tiagojsag) [inline app/styles/index.scss]: good catch

File: app/src/components/Shared/LinkTo.jsx (hunk 1 above)
Comment (andresgnlez) [inline app/src/components/Shared/LinkTo.jsx]: This class we'll be used in the WHOLE project replacing current arrows for this component. That's the idea.

Comment (tiagojsag) [inline app/src/components/Shared/LinkTo.jsx]: then it should be named and documented according to it.

**Pull request merged**